### PR TITLE
Fix readdir() bug when a non-zero offset is specified in filler

### PR DESCRIPTION
**Pull request by rostislav**
Hi Nikolaus,

This is a backport of my earlier commit in master fixing the same bug.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -2,6 +2,10 @@ FUSE 2.9.8
 ==========
 
 * Fixed rename deadlock on FreeBSD.
+* Added a test of `seekdir` to test_syscalls.
```
Comment (Nikratio) [inline ChangeLog]: 2.9.8 has already been released, so this needs a new entry ("Unreleased Changes")

**Comment by rostislav** [comment]
When updating the changelog as per your request, I noticed that the changes actually included in 2.9.8 are under "Unreleased Changes" in the changelog (e. g. fusermount hardening). I guess it's not intended?

**Comment by Nikratio** [comment]
Indeed, I forgot to commit this file. Should be fixed now. thanks for the pointer!

**Comment by rostislav** [comment]
Updated my commit with your latest update of `fuse_2_9_bugfix`, now my changes in ChangeLog are under "Unreleased Changes"

**Pull request merged**